### PR TITLE
Syn 717

### PR DESCRIPTION
**Pull request by mgebala8**
Implementation ab test (registration view)

### CODE REVIEW

File: .gatsby-context.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+module.exports = function (callback) {
+  var context = require.context('./pages', true, /(coffee|cjsx|ts|tsx|jsx|js|md|rmd|mkdn?|mdwn|mdown|markdown|litcoffee|ipynb|html|json|yaml|toml)$/); // eslint-disable-line
+  if (module.hot) {
+    module.hot.accept(context.id, function () {
+      context = require.context('./pages', true, /(coffee|cjsx|ts|tsx|jsx|js|md|rmd|mkdn?|mdwn|mdown|markdown|litcoffee|ipynb|html|json|yaml|toml)$/); // eslint-disable-line
+      return callback(context);
+    });
+  }
+  return callback(context);
+};
```
Comment (Idered) [inline .gatsby-context.js:15]: What is this file for?

File: data-pricing-plans.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"options":{"cbx":["3","5","10","25","50","100","175","250"],"api":["6","20","40","80","150","350","600","900","1250"]},"name":"paid-commitment","links":{"subscribe":"/v1.1/billing/plans/paid-commitment/subscribe/","self":"/v1.1/billing/plans/paid-commitment/"},"pricing":{"cbx":{"3":{"overage":"0.0000667","included":50000},"5":{"overage":"0.0000185","included":270000},"10":{"overage":"0.0000167","included":600000},"25":{"overage":"0.0000160","included":1560000},"50":{"overage":"0.0000152","included":3300000},"100":{"overage":"0.0000145","included":6900000},"175":{"overage":"0.0000136","included":12900000},"250":{"overage":"0.0000124","included":20100000}},"api":{"6":{"overage":"0.00003","included":200000},"20":{"overage":"0.00002","included":1000000},"40":{"overage":"0.00002","included":2000000},"80":{"overage":"0.0000178","included":4500000},"150":{"overage":"0.0000167","included":9000000},"350":{"overage":"0.0000152","included":23000000},"600":{"overage":"0.0000143","included":42000000},"900":{"overage":"0.0000129","included":70000000},"1250":{"overage":"0.0000125","included":100000000}}}}
```
Comment (Idered) [inline data-pricing-plans.json:1]: What is this file for?

File: data-tweets.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{"errors":true}
```
Comment (Idered) [inline data-tweets.json:1]: What is this file for?

File: src/components/shared/auth/signup.js [hunk 4]
```diff
@@ -1,22 +1,32 @@
-import {connect} from 'zefir/utils'
+import { connect } from 'zefir/utils'
```
Comment (Idered) [inline src/components/shared/auth/signup.js:1]: Don't add spacing in brackets.

File: src/components/shared/auth/signup.js [hunk 5]
```diff
@@ -1,22 +1,32 @@
...
 import Input from '../../ui/input'
 import InputList from '../../ui/input-list'
 import Style from './components/style'
-// import SocialButtons from './components/social-buttons'
+import SocialButtons from './components/social-buttons'
+
+const SignUpForm = ({email, password, toggleModal, register, messages,social}) => {
+
+return (
+
```
Comment (Idered) [inline src/components/shared/auth/signup.js:11]: Useless new line. Remove it

File: src/components/shared/auth/signup.js [hunk 6]
```diff
@@ -1,22 +1,32 @@
...
+  <form className='AuthForm' id='AuthForm' onSubmit={register}>
+
```
Comment (Idered) [inline src/components/shared/auth/signup.js:13]: Remove new line

File: src/components/shared/auth/signup.js [hunk 7]
```diff
@@ -1,22 +1,32 @@
...
+    { social && (
```
Comment (Idered) [inline src/components/shared/auth/signup.js:14]: No spaces in brackets

File: src/components/shared/auth/signup.js [hunk 8]
```diff
@@ -1,22 +1,32 @@
...
-      <SocialButtons />
-    </div> */}
-    {/* <div className='AuthForm__separator' /> */}
     <div className='AuthForm__column AuthForm__column--form'>
       <InputList errors={messages.get('auth.register')}>
         <Input full {...email} />
         <Input full {...password} />
       </InputList>
-      <Button full secondary>Create your account</Button>
+      <Button type="button" full secondary onClick={ () => { window.analytics.track('Sign up Website Confirmed'); document.getElementById("AuthForm").submit(); }}>Create your account</Button>
```
Comment (Idered) [inline src/components/shared/auth/signup.js:29]: Check how it's done in other places, like here https://github.com/Syncano/syncano.com/blob/syncano-ascend/src/components/shared/auth/signin.js

File: src/components/ui/modal.js [hunk 9]
```diff
@@ -224,4 +225,4 @@ Modal.init = ({
   }
 }
 
-export default connect(Modal)
+export default connect( Modal )
```
Comment (Idered) [inline src/components/ui/modal.js:228]: No spaces in brackets

File: src/components/ui/modal.js [hunk 10]
```diff
@@ -2,6 +2,7 @@ import {Component} from 'react'
 import {connect} from 'zefir/utils'
 
 class Modal extends Component {
+
```
Comment (Idered) [inline src/components/ui/modal.js:5]: Remove empty line

File: src/components/shared/header/user-nav.js [hunk 11]
```diff
@@ -5,6 +5,7 @@ const UserNav = ({theme, toggleSignUp, toggleSignIn, pageStatus}) => (
   <div className={`User-nav t-${theme}`}>
     <MatchAsGuest component={() => (
       <Nav>
+
```
Comment (Idered) [inline src/components/shared/header/user-nav.js:8]: Remove empty line

File: src/components/ui/page/index.js [hunk 12]
```diff
@@ -25,6 +27,30 @@ export default ({children}) => (
     </Modal>
 
     <Modal
+      name='signupver2'
+      title=''
```
Comment (Idered) [inline src/components/ui/page/index.js:31]: You don't have to pass title or subtitle if it's empty

File: src/components/ui/page/index.js [hunk 13]
```diff
@@ -57,6 +83,16 @@ export default ({children}) => (
         line-height: 1.25;
       }
 
+      .left {
```
Comment (Idered) [inline src/components/ui/page/index.js:86]: Move this and signupver2 jsx to separate file.

File: src/components/ui/page/index.js [hunk 14]
```diff
@@ -57,6 +83,16 @@ export default ({children}) => (
...
+        float: left;
+        width: 40%;
+      }
+
+      .right {
+        float: left;
+        width: 60%;
```
Comment (Idered) [inline src/components/ui/page/index.js:93]: `width: 60%` on class named `right`? No logic here. Change name of class.

File: src/components/ui/sign-description.js [hunk 15]
```diff
@@ -0,0 +1,65 @@
+const SignDescription = () => (
+  <div>
+    <div className="imageWrapper">
```
Comment (Idered) [inline src/components/ui/sign-description.js:3]: Check the nomenclature of css classes in other files

File: src/components/ui/sign-description.js [hunk 16]
```diff
@@ -0,0 +1,65 @@
...
+      <img src="https://media.licdn.com/mpr/mpr/shrinknp_200_200/p/5/005/084/138/2e66790.jpg" />
+    </div>
+    <span>We managed to build, launch and succeed with our Pokemon Go map app in just 48 hours of programming.
+    <author>Artur Czemiel<span>developer & founder, Aexol</span></author>
+    </span>
+
+    <style jsx>{`
+      .imageWrapper {
+        width:48px;
```
Comment (Idered) [inline src/components/ui/sign-description.js:12]: Use space after `:`

File: src/components/ui/sign-description.js [hunk 17]
```diff
@@ -0,0 +1,65 @@
...
+
+      .imageWrapper img{
+        width:48px;
+        height:48px;
+      }
+
+      span{
+        display:block;
+        margin:0px;
+	      width: 205px;
```
Comment (Idered) [inline src/components/ui/sign-description.js:28]: Install http://editorconfig.org/ in your editor. Don't mix tabs and spaces. Keep indentation consistent.

File: src/components/ui/sign-description.js [hunk 18]
```diff
@@ -0,0 +1,65 @@
...
+      }
+
+      .imageWrapper img{
+        width:48px;
+        height:48px;
+      }
+
+      span{
+        display:block;
+        margin:0px;
```
Comment (Idered) [inline src/components/ui/sign-description.js:27]: No need for px on `0`

File: src/components/ui/sign-description.js [hunk 19]
```diff
@@ -0,0 +1,65 @@
...
+        display:block;
+        text-align:left;
+  	    width: auto;
+  	    height: 18px;
+      	opacity: 0.8;
+      	font-family: Roboto;
+      	font-size: 16px;
+      	line-height: 1.12;
+      	color: #3e434a;
+        margin-top:20px;
```
Comment (Idered) [inline src/components/ui/sign-description.js:46]: Keep indentation consistent.

File: src/components/ui/sign-description.js [hunk 20]
```diff
@@ -0,0 +1,65 @@
+const SignDescription = () => (
+  <div>
+    <div className="imageWrapper">
+      <img src="https://media.licdn.com/mpr/mpr/shrinknp_200_200/p/5/005/084/138/2e66790.jpg" />
+    </div>
+    <span>We managed to build, launch and succeed with our Pokemon Go map app in just 48 hours of programming.
+    <author>Artur Czemiel<span>developer & founder, Aexol</span></author>
```
Comment (Idered) [inline src/components/ui/sign-description.js:7]: Where did you found that `author` tag?

File: src/components/ui/sign-description.js [hunk 21]
```diff
@@ -0,0 +1,65 @@
...
+      .imageWrapper {
+        width:48px;
+        height:48px;
+        border-radius:24px;
+        overflow:hidden;
+        margin-bottom:24px;
+        margin-top:20px;
+      }
+
+      .imageWrapper img{
```
Comment (Idered) [inline src/components/ui/sign-description.js:20]: Learn BEM syntax: https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/ and namespaces https://csswizardry.com/2015/03/more-transparent-ui-code-with-namespaces/

File: src/index.html [hunk 22]
```diff
@@ -7,6 +7,7 @@
     <meta key='og:site_name' property='og:site_name' content='Syncano' />
     <meta key='og:description' property='og:description' content='Don’t rebuild what has already been built. Syncano has hundreds of ready to use functions for your backend. Set up a scalable app in the cloud in just 45 seconds.' />
     <meta key='og:type' property='og:type' content='website' />
+    <script src=“https://cdn.optimizely.com/js/8415101584.js“></script>
```
Comment (Idered) [inline src/index.html:10]: Why did you add optimizely? It's already on devel https://github.com/Syncano/syncano.com/commit/94616ca159ddb37feebe7de0e05ea2be7de95755

Also you used bad quotation mark.

File: src/views/landing/index.js [hunk 23]
```diff
@@ -184,10 +184,13 @@ const Landing = ({
 Landing.init = ({
   services: {ui: {toggleModal}}
 }) => ({
-  toggleSignUpModal: () => {
+  toggleSignUpModal: (e) => {
+
```
Comment (Idered) [inline src/views/landing/index.js:188]: Remove blank line

File: src/views/signin.js [hunk 24]
```diff
@@ -23,7 +23,7 @@ const SignIn = () => (
 
     <Footer />
 
-    <style jsx>{`
+    <style jsx> {`
```
Comment (Idered) [inline src/views/signin.js:26]: No whitespace before `{`

File: data-tweets.json (hunk 3 above)
Comment (mgebala8) [reply data-tweets.json:1]: I do not know, I think it was updated automatically

File: data-pricing-plans.json (hunk 2 above)
Comment (mgebala8) [reply data-pricing-plans.json:1]: I do not know, I think it was updated automatically

File: .gatsby-context.js (hunk 1 above)
Comment (mgebala8) [reply .gatsby-context.js:15]: I do not know, I think it was updated automatically

File: data-pricing-plans.json (hunk 2 above)
Comment (23doors) [reply data-pricing-plans.json:1]: That's data for our pricing plans, how can it be updated automatically.

**Comment by 23doors** [comment]
@mgebala8 Use meaningful PR names that do not require looking up task in jira. Also, if a PR is not critical, wait/request a review for example from @Idered. Especially when some changes are well, questionable.

**Comment by mkucharz** [comment]
@mgebala8 Please do not merge anything before someone accept it!
